src/client/components/Overview/Overview.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Overview.styles.css';
+import OverviewSearchMessagesBtn from './OverviewSearchMessagesBtn';
+
+function Overview() {
+  return (
+    <div className="overview">
+      <h3 className="chatTitle">Chats</h3>
+      <div className="btnAndProfile">
```

Comment:
btn -and-profile

Use this naming scheme for all classes, remember to update in css